### ISSUE TITLE
[Enhancement] WNIC 4pts Question

### ISSUE THREAD

**Issue by Vinceli2401** [7ue9ueue/A-Level-CS-Revision#1]
<img width="723" alt="image" src="https://user-images.githubusercontent.com/62593369/233223168-1f7b73e1-9c99-4e08-a15e-69436379e974.png">

Sometimes the WNIC is tested as a 4 mark question, would be helpful to extend some points here...
• Provide interface to wireless network
• … as an antenna
• Receives analogue radio waves
• … convert them to digital / binary
• Checks incoming transmissions for correct MAC / IP address
• … ignore transmissions not intended for it
• Encrypts / encodes the data
• Decrypts / decodes the data
• Takes digital/binary input and converts to analogue waves
• … sends the radio waves via the antenna

**Comment by 7ue9ueue** [comment]
Yes, WNIC could be a 4 mark question, but I think that was old past papers. Now these points are updated.
